### DISCUSSION TITLE
How to disable ctrl-click to paste

### DISCUSSION THREAD

**Discussion by Xilhion** [Eugeny/tabby#6243] [Q&A]
Hi,

I start to love this terminal app but i have a settings that make me crazy.

I use Tabby v1.0.176 on Windows 10  21H2.
Setting are set to : 
- Copy on select
- Right click to paste (long press for context menu)
- Keyboard CTRL-SHIFT-C to copy and CTRL-SHIFT-V to paste

However, it keeps pasting when i use ctrl-leftclick and it is messing with other features (like ctrl-shift drag)

I don't find a way to disable this.
It does not do that on any other app i use, be it other terminals or not.

Thanks

**Comment by christoferd** [comment]
CTRL-SHIFT-V paste does not work for me. Has tabby changed something since your post?

**Reply by blsz** [reply]
For me, ctrl-leftclick pastes but it also begins to drag a new selection from the position that I clicked on.   I'm not sure what is supposed to happen.

**Comment by mboeker** [comment]
I also would really like to stop ctrl-click pasting!

**Comment by HenryChow505** [comment]
Please follow the screenshot below to disable it.

![image](https://github.com/Eugeny/tabby/assets/134348195/504cb2e0-81c5-406f-afe0-572adfcb1cbc)

**Comment by wangzhe258369** [comment]
I found that pressing Ctrl + left mouse click in Tabby triggers the right mouse click Action. I don't know where this can be set (canceled).

**Reply by mariusionescu** [reply]
Indeed, it seems in Tabby CTRL+LEFTCLICK triggers RIGHTCLICK. Does anyone know how to cancel this behavior?
